variant_remapping_tools/reads_to_remapped_variants.py

```diff
@@ -22,48 +25,48 @@ def calculate_new_variant_definition(left_read, right_read, ref_fasta):
     old_ref = info[2]
     old_alts = info[3].split(',')
 
+    operations = []
     # Define new ref and new pos
     new_ref = fetch_bases(ref_fasta, left_read.reference_name, left_read.reference_end + 1,
-                          right_read.pos - left_read.reference_end)
+                          right_read.reference_start - left_read.reference_end)
     new_pos = left_read.reference_end + 1
 
-    # TODO: All th operation bellow should be recorded so they can be used for subsequent changes of the genotypes.
+    # TODO: All the operation bellow should be recorded so they can be used for subsequent changes of the genotypes.
     # 1. Handle reference strand change
     if not left_read.is_reverse and not right_read.is_reverse:
         # Forward strand alignment
         old_ref_conv = old_ref
         old_alt_conv = old_alts
+        operations.append('st=+')
     elif left_read.is_reverse and right_read.is_reverse:
         # Reverse strand alignment
         old_ref_conv = reverse_complement(old_ref)
         old_alt_conv = [reverse_complement(alt) for alt in old_alts]
-    else:
-        error_msg = (f'Impossible read configuration: '
-                     f'read1 is_reverse: {left_read.is_reverse}, '
-                     f'read2 is_reverse: {right_read.is_reverse}, '
-                     f'read1 position: {left_read.pos}, '
-                     f'read2 position: {right_read.pos}')
-        print(error_msg)
-        raise ValueError(error_msg)
```

Comment:
Do we not need to track this case at all anymore?  (I agree with [this comment](https://github.com/EBIvariation/variant-remapping/pull/21#discussion_r600133586) about not raising an exception, maybe I've missed some further discussion on this topic...)

Reply:
This case is now filtered in the `pass_aligned_filtering` function so it does not need to be raised here

Reply:
I'd like to file a dissenting opinion: I think it's still worth checking for this condition here, _even if_ it's being already checked somewhere upstream. An additional assert here costs next to nothing, but it provides an additional line of defence in case something breaks upstream or in the tests

Reply:
Alright, that's fair enough. I've reinserted the `raise`